06_memory/module/dump.log

```diff
@@ -0,0 +1,92 @@
+[  298.814219]          512    0.000001633        0.000000467
+[  298.814246]         1024    0.000002050        0.000000850
+[  298.814274]         2048    0.000002950        0.000000793
+[  298.814303]         4096    0.000004512        0.000000458
+[  298.814337]         8192    0.000008435        0.000000887
+[  298.814386]        16384    0.000021068        0.000002765
+[  298.814448]        32768    0.000034193        0.000001396
+[  298.814541]        65536    0.000063922        0.000002658
+[  298.814748]       131072    0.000176155        0.000003162
+[  298.815827]       262144    0.001035695        0.000007956
+[  298.816375]       524288    0.000484085        0.000006352
+[  298.817431]      1048576    0.001013627        0.000007309
+[  298.820017]      2097152    0.002529880        0.000018653
+[  298.824176]      4194304    0.004050676        0.000025106
+[  298.824268]
+[  298.824268] ***** VMALLOC *****
+[  298.824309]  Buffer size  allocation time  freeing time
+[  298.825098]            2    0.000728726        0.000037023
+[  298.825176]            4    0.000008745        0.000005507
+[  298.825217]            8    0.000006048        0.000004090
+[  298.825252]           16    0.000005131        0.000003777
+[  298.825286]           32    0.000004857        0.000003510
+[  298.825319]           64    0.000004688        0.000003347
+[  298.825352]          128    0.000004566        0.000003571
+[  298.825384]          256    0.000004341        0.000003325
+[  298.825416]          512    0.000004227        0.000003293
+[  298.825448]         1024    0.000004132        0.000003450
+[  298.825480]         2048    0.000004412        0.000003236
+[  298.825512]         4096    0.000004433        0.000003299
+[  298.825550]         8192    0.000010074        0.000004023
+[  298.826268]        16384    0.000009264        0.000680626
+[  298.826377]        32768    0.000032110        0.000015596
+[  298.826437]        65536    0.000018238        0.000011864
+[  298.826515]       131072    0.000034054        0.000016950
+[  298.827182]       262144    0.000572216        0.000058913
+[  298.827453]       524288    0.000135317        0.000069280
+[  298.828604]      1048576    0.000983626        0.000119310
+[  298.829528]      2097152    0.000461145        0.000293261
+[  298.831590]      4194304    0.001454706        0.000571195
+[  298.835101]      8388608    0.002206364        0.001220781
+[  298.842035]     16777216    0.004284895        0.002591941
+[  298.855543]     33554432    0.008409522        0.004976955
+[  298.884494]     67108864    0.017163034        0.011618808
+[  298.884680]
+[  298.884680] ***** GET_FREE_PAGES *****
+[  298.884732]  Buffer size  allocation time  freeing time
+[  298.884902]         4096    0.000098796        0.000021249
+[  298.884941]         8192    0.000003362        0.000001051
+[  298.884981]        12288    0.000002121        0.000001024
```

Comment:
Is it possible to allocate exactly 12288 bytes using get_free_pages?

Reply:
no, wrong formula was used. Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/148/commits/7181b5d0c66692b76dc01a72d39a7497a761e93c